template/istqb.cls

```diff
@@ -53,6 +53,117 @@
 \usepackage{parskip}  % Put half of line height between paragraphs.
 \sloppy  % Prevent overfull lines.
 
+%% Language
+\PassOptionsToPackage{shorthands=off}{babel}
+\RequirePackage[czech,slovak,english]{babel}
+\ExplSyntaxOn
+\tl_new:N \g_istqb_language_tl
+\prop_new:N \g_istqb_reference_labels_prop
+\cs_new:Nn
+  \istqb_set_language:n
+  {
+    \tl_gset:Nn
+      \g_istqb_language_tl
+      { #1 }
+    \prop_gclear:N
+      \g_istqb_reference_labels_prop
+    \str_case:nnF
+      { #1 }
+      {
+        { cs } {
+          \selectlanguage { czech }
+          \gdef \istqbstandardsname { Standardy }
```

Comment:
Can we translate word `Appendix` as well which is added to the heading of each appendix automatically?

Reply:
I added the translation of "Appendix" in commit 63fa20c0b380fa3bc69db3d15aa95f31255732bb from this PR.